internal/bminventory/inventory.go

```diff
@@ -72,20 +72,21 @@ var (
 )
 
 type Config struct {
-	ImageBuilder         string            `envconfig:"IMAGE_BUILDER" default:"quay.io/ocpmetal/assisted-iso-create:latest"`
-	AgentDockerImg       string            `envconfig:"AGENT_DOCKER_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
-	ServiceBaseURL       string            `envconfig:"SERVICE_BASE_URL"`
-	ServiceCACertPath    string            `envconfig:"SERVICE_CA_CERT_PATH" default:""`
-	S3EndpointURL        string            `envconfig:"S3_ENDPOINT_URL" default:"http://10.35.59.36:30925"`
-	S3Bucket             string            `envconfig:"S3_BUCKET" default:"test"`
-	ImageExpirationTime  time.Duration     `envconfig:"IMAGE_EXPIRATION_TIME" default:"4h"`
-	AwsAccessKeyID       string            `envconfig:"AWS_ACCESS_KEY_ID" default:"accessKey1"`
-	AwsSecretAccessKey   string            `envconfig:"AWS_SECRET_ACCESS_KEY" default:"verySecretKey1"`
-	BaseDNSDomains       map[string]string `envconfig:"BASE_DNS_DOMAINS" default:""`
-	SkipCertVerification bool              `envconfig:"SKIP_CERT_VERIFICATION" default:"false"`
-	InstallRHCa          bool              `envconfig:"INSTALL_RH_CA" default:"false"`
-	RhQaRegCred          string            `envconfig:"REGISTRY_CREDS" default:""`
-	AgentTimeoutStart    time.Duration     `envconfig:"AGENT_TIMEOUT_START" default:"3m"`
+	ImageBuilder             string            `envconfig:"IMAGE_BUILDER" default:"quay.io/ocpmetal/assisted-iso-create:latest"`
+	AgentDockerImg           string            `envconfig:"AGENT_DOCKER_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
+	ServiceBaseURL           string            `envconfig:"SERVICE_BASE_URL"`
+	ServiceCACertPath        string            `envconfig:"SERVICE_CA_CERT_PATH" default:""`
+	S3EndpointURL            string            `envconfig:"S3_ENDPOINT_URL" default:"http://10.35.59.36:30925"`
+	S3Bucket                 string            `envconfig:"S3_BUCKET" default:"test"`
+	ImageExpirationTime      time.Duration     `envconfig:"IMAGE_EXPIRATION_TIME" default:"4h"`
+	AwsAccessKeyID           string            `envconfig:"AWS_ACCESS_KEY_ID" default:"accessKey1"`
+	AwsSecretAccessKey       string            `envconfig:"AWS_SECRET_ACCESS_KEY" default:"verySecretKey1"`
+	BaseDNSDomains           map[string]string `envconfig:"BASE_DNS_DOMAINS" default:""`
+	SkipCertVerification     bool              `envconfig:"SKIP_CERT_VERIFICATION" default:"false"`
+	InstallRHCa              bool              `envconfig:"INSTALL_RH_CA" default:"false"`
+	RhQaRegCred              string            `envconfig:"REGISTRY_CREDS" default:""`
+	AgentTimeoutStart        time.Duration     `envconfig:"AGENT_TIMEOUT_START" default:"3m"`
+	DeletedUnregisteredAfter time.Duration     `envconfig:"DELETED_UNREGISTERED_AFTER" default:"72h"`
```

Comment:
@avishayt @filanov 
does this interval make sense to delete soft deleted clusters and hosts?

Reply:
A week maybe? if someone starts looking it after the weekend